tsl/test/expected/dist_views.out

```diff
@@ -39,15 +39,22 @@ SELECT setseed(1);
  
 (1 row)
 
-CREATE TABLE dist_table(time timestamptz, device int, temp float);
+CREATE TABLE dist_table(time timestamptz, device int, temp float, timdim date);
```

Comment:
can you make time and timdim NOT NULL, this will get rid of the NOTICEs and remove some noise